ARM64 / Apple M1

Can't install this package on Apple's M1 Silicon.

The following error is produced:
```
% npm i -g @customrealms/cli 
npm ERR! code 1
npm ERR! path /opt/homebrew/lib/node_modules/@customrealms/cli
npm ERR! command failed
npm ERR! command sh -c node postinstall.js install
npm ERR! Installation is not supported for this architecture: arm64
npm ERR! Invalid inputs

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/[...]/.npm/_logs/2022-01-31T14_54_25_019Z-debug-0.log
```

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -11,6 +11,17 @@ builds:
       - darwin
     goarch:
       - amd64
+  - id: customrealms-cli-darwin-arm64
+    ldflags:
+      - -s -w -X github.com/customrealms/cli/pkg/version.Version={{.Version}}
+    binary: crx
+    main: ./cmd/crx/main.go
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - darwin
+    goarch:
+      - arm64
   - id: customrealms-cli-linux-amd64
     ldflags:
       - -s -w -X github.com/customrealms/cli/pkg/version.Version={{.Version}}

--- a/postinstall.js
+++ b/postinstall.js
@@ -9,7 +9,8 @@ const fs = require('fs');
 var ARCH_MAPPING = {
     "ia32": "386",
     "x64": "amd64",
-    "arm": "arm"
+    "arm": "arm",
+    "arm64": "arm64"
 };
 
 // Mapping between Node's `process.platform` to Golang's GOOS
@@ -137,7 +138,7 @@ async function install(callback) {
     var opts = parsePackageJson();
     if (!opts) return callback(INVALID_INPUT);
     mkdirp.sync(opts.binPath);
-    console.info(`Copying the relevant binary for your platform ${process.platform}`);
+    console.info(`Copying the relevant binary for your platform ${process.platform} (${process.arch})`);
 
     // Map the process platform and arch strings to the Go version
     const platform = PLATFORM_MAPPING[process.platform];

